ddtrace/contrib/internal/pytest/_atr_utils.py

```diff
@@ -50,11 +53,14 @@ class _QUARANTINE_ATR_RETRY_OUTCOMES(_ATR_RETRY_OUTCOMES):
 def atr_handle_retries(
     test_id: InternalTestId,
     item: pytest.Item,
-    when: str,
-    original_result: pytest_TestReport,
+    test_reports: t.Dict[str, pytest_TestReport],
     test_outcome: _TestOutcome,
     is_quarantined: bool = False,
 ):
+    setup_report = test_reports.get("setup")
+    call_report = test_reports.get("call")
+    teardown_report = test_reports.get("teardown")
```

Comment:
nit: I'd rather have these as constants, just to avoid any typos
```
   # Move these to constants and import them here
   SETUP = "setup"
   CALL = "call"
   TEARDOWN = "teardown"
   [...]
```

```suggestion
    setup_report = test_reports.get(SETUP)
    call_report = test_reports.get(CALL)
    teardown_report = test_reports.get(TEARDOWN)
```

Reply:
Done https://github.com/DataDog/dd-trace-py/pull/13376/commits/0d98f571412f63e4a880a48f8320f207b5fbfac6